joomla/joomla-cms
# Make including pw in mail to user optional when created by admin

Pull Request for Issue #10376  .
#### Summary of Changes

When creating user accounts as administrator and sending an email, it is now optional to include the password in the mail.
This is configurable on the plugin. Sending the email was already optional. 
When sending emails, including the password in the email is now optional.
#### Testing Instructions

Apply this patch
As administrator create an user account.
Verify that the password is included in the mail.
Configure plugin User - Joomla to exlude the password.
Create an user account.
Verify that the password in not in the mail.

This closes issue https://github.com/joomla/joomla-cms/issues/10376
